docs/source/conf.py

```diff
@@ -275,6 +275,7 @@ def process_readme():
         readme = file.read()
 
     readme = readme.replace("# eo-learn", "# Introduction").replace("docs/source/", "")
+    readme = readme.replace("[`", "[").replace("`]", "]")
```

Comment:
why do we do this?

Reply:
There are issues in some of the docs, namely here: https://eo-learn.readthedocs.io/en/latest/#more-examples

![image](https://user-images.githubusercontent.com/37869250/220589499-7bd359d0-e2c1-42c1-a4c1-df29676fb9fc.png)

It is because backticks are used in the url name template. This removes the backticks. It works in markdown, but not in the linked docs. We could just remove the backticks, but I noticed the same rule is applied also in `eo-grow`.

Reply:
ah, i see. consider this resolved